### ISSUE TITLE
Error when duration is set to 0s

### ISSUE THREAD

**Issue by erihanse** [nakabonne/ali#82]
If I run
```
➜  ~ ali --duration=0s https://google.com
duration must be greater than 0
```
I get that duration must be greater than 0. But in the help text it says:
```
  -d, --duration duration   The amount of time to issue requests to the targets. Give 0s for an infinite attack. (default 10s)
```
Is infinite attack supported or not?

**Comment by nakabonne** [comment]
Hey, thank you for reporting! I've just fixed it (https://github.com/nakabonne/ali/pull/84) and will be available at the next release.

**Comment by nakabonne** [comment]
@erihanse You can now available at [v0.5.3](https://github.com/nakabonne/ali/releases/tag/v0.5.3)!
